### DISCUSSION TITLE
Layout session - Octavia Nuxt Admin Pro

### DISCUSSION THREAD

**Discussion by israelgs21** [uilibrary/Octavia-admin-Pro-Issues#23] [Q&A]
Thanks

I have Octavia Nuxt Admin Pro and I want to place the SignInTwo page alone with the Layout empty, is this possible?

**Comment by naime-hossain** [comment]
> Thanks
> 
> 
> I have Octavia Nuxt Admin Pro and I want to place the SignInTwo page alone with the Layout empty, is this possible?
> 
> 

Can you please explain your issue in details. Thank you

**Reply by israelgs21** [reply]
I want to place the login as the home page, I attach the image, thank you.
![Index](https://user-images.githubusercontent.com/86500888/133719653-f09487e2-2a8a-4f45-ae4a-34fa6d004f3c.png)

**Reply by naime-hossain** [reply]
Edit this file src/router/index.js
![image](https://user-images.githubusercontent.com/12586039/133796618-ef3a7798-5da0-413f-980c-5fa7b4e51c95.png)

replace this 


```
      path: '/',
      component: () => import('@/views/app/Index'),
      beforeEnter: authenticate,
      redirect: '/app/dashboard/analytic',
```

with this 

```
      path: '/',
      component: () => import('@/views/app/Index'),
      beforeEnter: authenticate,
      redirect: 'app/sessions/sign-in',
```

**Reply by israelgs21** [reply]
Thank you, sorry, it seems to me that you tell me about the Octavia Admin Pro v2.5.3 project and I have the Octavia-NuxtJs-v1.2.0 project, in the latter you could support me.

**Reply by israelgs21** [reply]
[Octavia-NuxtJs-v1.2.0](https://store.vuetifyjs.com/products/octavia-nuxt)
Thank you
